vaadin-notification-flow-parent/vaadin-notification-flow/src/main/java/com/vaadin/flow/component/notification/Notification.java

```diff
@@ -130,8 +131,6 @@ static Position fromClientName(String clientName) {
      */
     public Notification() {
         initBaseElementsAndListeners();
-        getElement().getNode().runWhenAttached(ui -> ui
-                .beforeClientResponse(this, context -> deferredJob.accept(ui)));
```

Comment:
This is not needed anymore as the consumer is now always run in the attach handler.